project/cmake/modules/FindOpenGLES.cmake

```diff
@@ -1,51 +1,48 @@
-# - Try to find OpenGLES2
-# Once done this will define
+#.rst:
+# FindOpenGLES
+# ------------
+# Finds the OpenGLES2 library
 #
-#  OPENGLES_FOUND        - system has OpenGLES2
-#  OPENGLES_INCLUDE_DIR  - the GLES2 include directory
-#  OPENGLES_LIBRARIES    - Link these to use OpenGLES2
+# This will will define the following variables::
+#
+# OPENGLES_FOUND - system has OpenGLES
+# OPENGLES_INCLUDE_DIRS - the OpenGLES include directory
+# OPENGLES_LIBRARIES - the OpenGLES libraries
+# OPENGLES_DEFINITIONS - the OpenGLES definitions
 
 find_package(EMBEDDED)
-find_package(PkgConfig)
 
 if(PKG_CONFIG_FOUND AND NOT PLATFORM STREQUAL "raspberry-pi")
-  pkg_check_modules(OPENGLES glesv2)
-  if(NOT OPENGLES_FOUND AND NOT EMBEDDED_FOUND STREQUAL "EMBEDDED_FOUND-NOTFOUND")
+  pkg_check_modules(PC_OPENGLES glesv2)
+  if(NOT OPENGLES_FOUND AND EMBEDDED_FOUND)
```

Comment:
you mean the strequal? I tested it on the host by setting the vars so that I end up in the gles/embedded branch. Together with an empty `/opt/vc/include/bcm_host.h` I was able to verify both cases EMBEDDED_FOUND and NOT EMBEDDED_FOUND. Works well and is also according to the doc: https://cmake.org/cmake/help/v3.0/command/if.html?highlight=#command:if.